provision-contest/ansible/Makefile

```diff
@@ -30,6 +30,9 @@ endif
 domserver judgehost admin grafana cds scoreboard mgmt: %: %.yml hosts group_vars/all/secret.yml $(LIBVENDOR) $(SSHKEY) $(SSHKEY).pub
 	ansible-playbook -i hosts $<
 
+codeonly-%:
```

Comment:
```suggestion
CODEONLY_ROLES=$(addprefix codeonly-,$(ROLES))
$(CODEONLY_ROLES): codeonly-%:
```

Reply:
This would now make that we also get codeonly for admin (where it would work) and for cds/scoreboard/mgmt/grafana which don't have the domjudge_* roles.

Reply:
Ah, yes. So then maybe the `codeonly-` prefix should only be added to the roles where it makes sense.